src/main/java/nextstep/subway/auth/application/AuthService.java

```diff
@@ -33,13 +32,26 @@ public TokenResponse login(TokenRequest request) {
     }
 
     @Transactional(readOnly = true)
-    public LoginMember findMemberByToken(String credentials) {
-        if (!jwtTokenProvider.validateToken(credentials)) {
-            throw new AuthorizationException();
+    public LoginMember findMemberByToken(String credentials, boolean isRequired) {
+        boolean isValidToken = isValidToken(credentials);
+
+        isTokenRequired(isRequired, isValidToken);
+        if (!isValidToken) {
+            return LoginMember.ofByNotLogin();
```

Comment:
```suggestion
            return LoginMember.ofNotLogin();
```
By를 쓰지 않고 쓰는 편이 자연스러워보이기도 하는데 어떠신가요..?

Reply:
오옹 네넵 더 자연스러운것 같습니다...!  👍 감사합니다